wtransport/Cargo.toml

```diff
@@ -31,12 +31,12 @@ required-features = ["self-signed"]
 [dependencies]
 bytes = "1.4.0"
 pem = "3.0.4"
-quinn = "0.10.1"
+quinn = { version = "0.11.3", default-features = false, features = ["runtime-tokio", "rustls"] }
 rcgen = { version = "0.13.1", optional = true }
-rustls = { version = "0.21.1", features = ["dangerous_configuration"] }
+rustls = { version = "0.23.12", default-features = false, features = ["ring"] }
```

Comment:
`quinn-proto` always uses `ring` so we can too https://github.com/quinn-rs/quinn/blob/5e0d96b76793c5c5dec15e24976c44e70d34c0b8/quinn-proto/Cargo.toml#L19